resolves #648 refresh documentation (#649)
asciidoctor/asciidoctor-browser-extension: docs/modules/ROOT/pages/install.adoc

= Install

> You can install the extension from your browser's web-store, or a local copy of the extension app.

Reviewer: Do we need to repeat "from"? Also, extension app might be redundant?

```suggestion
You can install the extension from your browser's web-store, or from a local copy of the extension.
```

> == Install from web-store

Reviewer: I think we should use "Web Store" (not "web-store")

Reviewer: Or "extensions store" ?

Author: Good suggestions. Perhaps "Web Store" would be the more informative term. Extension Store sounds more like a repository than the official source of extensions for the browser.

Author: Fixed by https://github.com/asciidoctor/asciidoctor-browser-extension/pull/649/commits/091332bded398a274fe680b17a8d80e1a941fbe0

> To install the Extension from your browser's web-store:

Reviewer: The sentence might be a bit convoluted. I prefer direct and short sentences:

```suggestion
Install the Extension on your browser using the image links below:
```

And maybe we can add a `TIP: Alternatively, you can search "Asciidoctor.js Live Preview" in your Browser's Web Store.`

What do you think?

+

> [%autowidth,cols=2*a]

Reviewer: ```suggestion
[%autowidth,cols=2*a,grid=none]
```

+
[%autowidth,cols=2*a]
|===
|[caption=]
.Chrome
image::chrome-webstore-badge.png["Image link to the Chrome, Chromium or Brave web-store",212,64,link={url-chrome-webstore}]

|[caption=]
.Firefox
image::firefox-webstore-badge.svg[Image link to the Firefox web-store,184,64,link={url-mozilla-addon}]

|[caption=]
.Opera
image::opera-webstore-badge.png[Image link to the Opera web-store,227,64,link={url-opera-addon}]

|[caption=]
.Edge
image::edge-webstore-badge.png[Image link to the Microsoft Edge web-store,177,64,link={url-edge-webstore}]
|===

. In order to preview local files on {chromium-browser-list} browsers,
you will need to grant filesystem access to the extension:

.. Type the URL `chrome://extensions` into the address bar and press kbd:[Enter]
.. Type `asciidoctor` into the extension search bar then when the {project-name} block is visible, click its btn:[Details] button... Check `Allow access to file URLs`

. You have now installed the extension, and can open local or remote *.adoc*, *.ad*, *.asc*, *.asciidoc* files in your browser
+
Enjoy!

[NOTE]
====

> If you are using a Chromium browser, and want to view local files, remember to enable `Allow access to file URLs` above.

Reviewer: I feel like this is unnecessary since we already explain that just a few lines above ☝🏻 

"In order to preview local files on ... browsers, you will need to grant filesystem access to the extension:"

Author: From experience, this is the number one problem with Chrome installations. To not repeat ourselves, perhaps we say it once with a NOTE admonition the first time, and remove this second reminder note.

Author: Fixed by https://github.com/asciidoctor/asciidoctor-browser-extension/pull/649/commits/66d5cb7da94ef1fa91c2b48294dd4ccadbfc901d

====

NOTE: There are known issues with Firefox. See xref:firefox-known-issues.adoc[].

== Local installations

Local installations are for users and developers when:

* Your machine does not have access to the internet or extension stores
* You are testing an unreleased (locally built) version

> Loading your local copy of the extension on {chromium-browser-list} or Firefox is super easy!

Reviewer: I believe we shouldn't use "super easy" in documentation. It can be frustrating if, for some reasons, it's not super easy for you 😄

Author: I do agree. It irks me reading documentation that says "simply...." as people approach it from both foundation and advanced knowledge levels.

Author: Resolved by https://github.com/asciidoctor/asciidoctor-browser-extension/pull/649/commits/9ae1bfab173da12a0a5e094bee05dab3c6ee4cb6
